opentelekomcloud/resource_opentelekomcloud_cce_node_v3.go

```diff
@@ -324,10 +324,19 @@ func resourceCCENodeV3Create(d *schema.ResourceData, meta interface{}) error {
 	// eip_count and bandwidth_size parameters must be set simultaneously
 	bandwidthSize := d.Get("bandwidth_size").(int)
 	eipCount := d.Get("eip_count").(int)
+	ipType := d.Get("iptype").(string)
+	shareType := d.Get("sharetype").(string)
+	chargeMode := d.Get("bandwidth_charge_mode").(string)
 	if bandwidthSize > 0 && eipCount == 0 {
 		eipCount = 1
 	}
 
+	if eipCount == 1 && (ipType == "" || shareType == "" || chargeMode == "") {
```

Comment:
All values are overwritten if even one of them is empty. You can just define default values for schema